core/Plugin/Report.php

```diff
@@ -495,19 +495,21 @@ protected function getMetricsDocumentation()
 
         $processedMetrics = $this->processedMetrics ?: array();
         foreach ($processedMetrics as $processedMetric) {
-            if (!($processedMetric instanceof ProcessedMetric)) {
-                continue;
-            }
-
-            $name = $processedMetric->getName();
-            $metricDocs = $processedMetric->getDocumentation();
-            if (empty($metricDocs)) {
-                $metricDocs = @$translations[$name];
+            if (is_string($processedMetric) && !empty($translations[$processedMetric])) {
+                $documentation[$processedMetric] = $translations[$processedMetric];
```

Comment:
I had to replace the Processed metric object in `Goals.get` report with a string 'conversion_rate'. This caused the metric documentation for this column was no longer in the processed report. Therefore I added that it will also translate a string column if possible. We do now return more metrics documentations which is good since the 4 default processed metrics are defined as strings as well and there was no documentation available for them.

Unfortunately 2 or 3 reports (`GetCountry`, `GetWebsites`) seem to sometime now contain a `conversion_rate` documentation although it does not always contain a `conversion_rate`. I think it is still ok to show that documentation for now as the column sometimes seems to be present. Needs to be fixed in the actual report.